continue-reading.html

```diff
@@ -1,13 +1,155 @@
 <!DOCTYPE HTML>
+<link href="continuereading.css" rel="stylesheet" />
+<link href="https://fonts.googleapis.com/css2?family=Peddana&display=swap" rel="stylesheet">
+
 <html>
-  <head>
-    <title>Continue Reading</title>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
-  </head>
-  <body>
-    <section id="header"></section>
-    <section id="main"></section>
-    <section id="footer"></section>
-  </body>
-</html>
\ No newline at end of file
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+</head>
+
+<body>
+  <!--Content Start and Title...-->
+  <a href="index.html" class="continuetitle"> Let's Read More About This!</a>
+  <h2>I guess If You Really Want Too...</h2>
+  <hr class="linebrk">
+  <hr class="linebrk1">
+  <!--Above picture text and first picture/first screenshot of continue page-->
+  <section class="pagestart">

```

Comment:
You have this `section` with a class on it, yet you're not using it to help style your page. This could probably be a good place to start to refactor